src/patternfly/layouts/Bullseye/styles.scss

```diff
@@ -1,8 +1,18 @@
+@import "../../patternfly-utilities";
+
+$pf-l-bullseye--Padding: 0 !default;
+$pf-l-bullseye__item--Padding: 0 !default;
+
 .pf-l-bullseye {
```

Comment:
@srambach Not changes here, but just a question for the future. Does bullseye item needs a width? if you add more text this will just stretch to the size of it's container.
Maybe a modifier class for sm, md and lg ?

Reply:
What you you define those as then?